.github/workflows/smoke-test-daily.yml

```diff
@@ -242,49 +216,39 @@ jobs:
               run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js upload-plugin.spec.js
 
             - name: Run the rest of E2E tests
-              id: e2e
               working-directory: plugins/woocommerce
               env:
                   E2E_MAX_FAILURES: 15
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js basic.spec.js
+              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js
 
             - name: Generate E2E Test report.
-              id: report
-              if: |
-                  always() &&
-                  (
-                    steps.e2e-upload.conclusion != 'cancelled' ||
-                    steps.e2e-upload.conclusion != 'skipped' ||
-                    steps.e2e.conclusion != 'cancelled' ||
-                    steps.e2e.conclusion != 'skipped' 
-                  )
+              if: success() || failure()
               working-directory: plugins/woocommerce
               run: pnpm exec allure generate --clean ${{ env.ALLURE_RESULTS_DIR }} --output ${{ env.ALLURE_REPORT_DIR }}
 
             - name: Archive E2E test report
-              if: |
-                  always() &&
-                  steps.report.conclusion == 'success'
+              if: success() || failure()
               uses: actions/upload-artifact@v3
               with:
-                  name: Smoke tests with ${{ matrix.plugin }} plugin installed (run ${{ github.run_number }})
+                  name: Smoke tests on trunk with ${{ matrix.plugin }} plugin installed (run ${{ github.run_number }})
                   path: |
                       ${{ env.ALLURE_RESULTS_DIR }}
                       ${{ env.ALLURE_REPORT_DIR }}
                   if-no-files-found: ignore
                   retention-days: 5
 
     trunk-results:
```

Comment:
Hi Tam! I initially thought of naming it to "nightly" too, but it seems that the Puppeteer version of our "Smoke tests with X plugin installed" tests are actually being ran on `trunk` instead of on the nighty build. So I did the same here on Playwright.

This is the latest version of the [`smoke-test-daily.yml`](https://github.com/woocommerce/woocommerce/blob/04b74c105361e5eca850d4f0ce02dc9f14556c3a/.github/workflows/smoke-test-daily.yml) file before the Playwright migration. According to its [Build zip](https://github.com/woocommerce/woocommerce/blob/04b74c105361e5eca850d4f0ce02dc9f14556c3a/.github/workflows/smoke-test-daily.yml#L117-L139) job, it seems that it checks out `trunk`, and then builds a zip from it. Afterwards, this zip will be extracted onto the `plugins/woocommerce` folder as shown [here](https://github.com/woocommerce/woocommerce/blob/04b74c105361e5eca850d4f0ce02dc9f14556c3a/.github/workflows/smoke-test-daily.yml#L178-L182).

So I thought maybe we should name these tests to `trunk` instead of nightly.